packages/wonder-blocks-toolbar/generated-snapshot.test.js

```diff
@@ -126,8 +142,6 @@ describe("wonder-blocks-toolbar", () => {
         const example = (
             <View style={style}>
                 <Toolbar color="dark" title="Title" />
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Sorry for not picking this up.  I keep forgetting that design that doesn't want anything in here.  It feels weird to have all this blank space.  🤷‍♂️

Reply:
No problem at all! I should have caught it earlier.